1_hello_world/a.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
what is this file? was this accidentally left in the commit or part of something else?